### PR TITLE
[enterprise-4.14]OSDOCS#9899: Add hosted control planes known issues

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-9899

Link to docs preview: https://73976--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-known-issues

QE review:
- [x] QE has approved this change.

Additional information: This PR is a part of CM

### CODE REVIEW

**Comment by davemulford** [comment]
LGTM

**Comment by xltian** [comment]
LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -2945,6 +2945,12 @@ To resolve those issues, configure the DNS server settings by following the guid
 
 * In hosted control planes on {VirtProductName}, worker nodes might lose network connectivity after a restart. (link:https://issues.redhat.com/browse/OCPBUGS-23208[*OCPBUGS-23208*])
 
+* In hosted control planes for {product-title}, the HyperShift Operator extracts the release metadata only once during Operator initialization. When you make changes in the management cluster or create a hosted cluster, the HyperShift Operator does not refresh the release metadata. As a workaround, restart the HyperShift Operator by deleting its pod deployment. (link:https://issues.redhat.com/browse/OCPBUGS-29110[*OCPBUGS-29110*])
+
+* In hosted control planes for {product-title}, when you create the custom resource definition (CRD) for `ImageDigestMirrorSet` and `ImageContentSourcePolicy` objects at the same time in disconnected environment, the HyperShift Operator creates the object only for the `ImageDigestMirrorSet` CRD ignoring the `ImageContentSourcePolicy` CRD.  As a workaround, copy the `ImageContentSourcePolicies` object configuration in the `ImageDigestMirrorSet` CRD. (link:https://issues.redhat.com/browse/OCPBUGS-29466[*OCPBUGS-29466*])
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:2950]:
```suggestion
* In hosted control planes for {product-title}, when you create the custom resource definition (CRD) for `ImageDigestMirrorSet` and `ImageContentSourcePolicy` objects at the same time in a disconnected environment, the HyperShift Operator creates the object only for the `ImageDigestMirrorSet` CRD, ignoring the `ImageContentSourcePolicy` CRD.  As a workaround, copy the `ImageContentSourcePolicies` object configuration in the `ImageDigestMirrorSet` CRD. (link:https://issues.redhat.com/browse/OCPBUGS-29466[*OCPBUGS-29466*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -2945,6 +2945,12 @@ To resolve those issues, configure the DNS server settings by following the guid
...
+
+* In hosted control planes for {product-title}, when creating a hosted cluster in disconnected environment, if you do not set the `hypershift.openshift.io/control-plane-operator-image` annotation explicitly in the `HostedCluster` resource, the hosted cluster deployment fails with an error. (link:https://issues.redhat.com/browse/OCPBUGS-29494[*OCPBUGS-29494*])
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:2952]:
```suggestion
* In hosted control planes for {product-title}, when creating a hosted cluster in a disconnected environment, if you do not set the `hypershift.openshift.io/control-plane-operator-image` annotation explicitly in the `HostedCluster` resource, the hosted cluster deployment fails with an error. (link:https://issues.redhat.com/browse/OCPBUGS-29494[*OCPBUGS-29494*])
```

**Review by adellape — COMMENTED** [review]
Added a few peer-review-style comments, but +1 for the CM ack (on DPM behalf as Kathryn is currently OoO).

**Review by adellape — APPROVED** [review]

**Pull request merged**